resources/docs/inertia/creating-chirps.md

```diff
@@ -425,22 +389,22 @@ Our form has been configured to post messages to the `chirps.store` route that w
 
 ```php filename=app/Http/Controllers/ChirpController.php
 <?php
-// [tl! collapse:start]
```

Comment:
Any reason this stuff was removed? @nunomaduro

Reply:
Because there is a new import, on the line 396 `use Illuminate\Http\RedirectResponse;`, I don't see the of collapsing only 3 lines.